app/src/main/java/ch/epfl/sweng/favors/utils/email/RetrofitClient.java

```diff
@@ -26,7 +26,7 @@
 
 import static java.net.HttpURLConnection.HTTP_OK;
 
-class RetrofitClient extends RetrofitDispatcher{
+public class RetrofitClient extends RetrofitDispatcher{
```

Comment:
sorry about this, probably we should consider using reflection
But kept getting access errors so went with this for now (are there any specific concerns with this Jeremey?)

Reply:
Well there is some issues with calling it properly. It is much better to have it in package private. What are the issues in particular?

Reply:
Where exactly did you get access errors with RetrofitClient ?

Reply:
## This is not OK. Read comment bellow about why